Source/CarthageKit/Availability.swift

```diff
@@ -45,36 +46,31 @@ public func duplicateProjectsInCartfiles(_ cartfile1: Cartfile, _ cartfile2: Car
 public enum ProjectIdentifier {}
 
 // MARK: - Git.swift
-
```

Comment:
Agreed. I mentioned this above, but it was collapsed. We should either require or allow a line after. I don't particularly care which.